### PR TITLE
Update license to current team & year

### LINKED ISSUE #678: Update license to current team & year
Currently [license](https://github.com/cake-build/cake/blob/develop/LICENSE) is so last year :wink:

### PR DESCRIPTION
**Pull request by devlead**
This fixes #678

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 - 2015 Patrik Svensson
+Copyright (c) 2014 - 2016 Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors
```
Comment (gep13) [inline LICENSE]: @devlead can we simply say 2014 - Present here?

**Comment by patriksvensson** [comment]
Maybe we can simply remove 2016 from the Copyright notice.

**Comment by devlead** [comment]
@gep13 @patriksvensson 
Perhaps just omit year entirely?
See [dotnet/corefx/LICENCE](https://github.com/dotnet/corefx/blob/master/LICENSE) as example:
image

**Comment by patriksvensson** [comment]
Yeah, that sounds good to me :+1:

**Comment by gep13** [comment]
@devlead I actually did the same check against the OLW license.  Removing it makes sense to me, and we have a precedence now to follow.

**Comment by devlead** [comment]
@gep13 @patriksvensson Adjusted and rebased.

**Pull request merged**